hera_cal/smooth_cal.py

```diff
@@ -182,15 +263,28 @@ def time_freq_2D_filter(gains, wgts, freqs, times, freq_scale=10.0, time_scale=1
                     and fringe rate = 0
             'plus': produce a separable calibration solution by only keeping modes with 0 delay,
```

Comment:
Explain that filter_mode `plus` is not implemented for DPSS.

Reply:
Added this